### PR TITLE
fix: get a timestamp not a timestamp offset by local time

### LINKED ISSUE #89: Incorrect timestamp for 'time'
The following code:
https://github.com/linz/python-linz-logger/blob/d332d79b6ce7e46f352bd7063d705b32798a6211/linz_logger/logger.py#L67
returns an incorrect timestamp. If you convert this `timestamp` into a `date`, the time does not match with the current time.

### PR DESCRIPTION
**Pull request by blacha**
Fixes #89

### CODE REVIEW

**Review by amfage — APPROVED** [review]
LGTM

**Review by paulfouquet — APPROVED** [review]
nice

File: linz_logger/test_logger.py [hunk 1]
```diff
@@ -28,3 +29,15 @@ def test_trace_at_debug_level(capsys):
...
+def test_timestamp(capsys):
+    """Test timestamp is actually a timestamp"""
+    set_level(LogLevel.trace)
+
+    systime = int(os.popen("date +%s").read()) * 1000
+    get_log().trace("abc")
+    stdout, _ = capsys.readouterr()
+    log = json.loads(stdout)
+
+    assert log["time"] - systime < 1000
```
Comment (l0b0) [inline linz_logger/test_logger.py:43]: Any kind of time-dependent test like this is going to be brittle. The common way to fix that would be to sample the time and dependency-injecting a static clock into `get_log` to verify that the timestamp is the same.

Comment (blacha) [reply linz_logger/test_logger.py:43]: if this test takes more than a second we have bigger problems.

Comment (l0b0) [reply linz_logger/test_logger.py:43]: In this case it might be fine - it's not like this is ever going to be a huge code base. But you'd be surprised.

**Pull request merged**